packages/statemanager/CHANGELOG.md

```diff
@@ -6,6 +6,30 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 2.1.0 - 2023-10-23
+
+### New Diff-Based Code Cache
+
+This release introduces a new code cache implementation, see PR [#3022](https://github.com/ethereumjs/ethereumjs-monorepo/pull/3022) and [#3080](https://github.com/ethereumjs/ethereumjs-monorepo/pull/3080). The new cache complements the expanded account and storage caches and now also tracks stored/deployed-code-changes along commits and reverts and so only keeps code in the cache which made it to the final state change.
+
+The new cache is substantially more robust towards various type of revert-based attacks and grows a more-used cache over time.
```

Comment:
```suggestion
The new cache is substantially more robust towards various types of revert-based attacks and grows a more-used cache over time.
```

Reply:
I'm not sure I understand what "grows a more-used cache over time" means, here. Is this meaning to say that the cache tracks changes to an account's byte code with constant memory-usage complexity?

Reply:
Yes, exactly that. 🙂 I might update this on occasion if it is a bit hard to grasp.